cfme/tests/intelligence/reports/test_validate_chargeback_report.py

```diff
@@ -0,0 +1,465 @@
+# -*- coding: utf-8 -*-
+
+import cfme.intelligence.chargeback as cb
+import cfme.configure.access_control as ac
+import fauxfactory
+import pytest
+import time
+
+from cfme import Credential
+from cfme.common.vm import VM
+from cfme.configure.configuration import get_server_roles, set_server_roles, candu
+from cfme.intelligence.reports.reports import CustomReport
+from datetime import date
+from utils import testgen, version
+
+
+def pytest_generate_tests(metafunc):
+    # Filter out providers not meant for Chargeback Testing
+    argnames, argvalues, idlist = testgen.infra_providers(metafunc, required_fields=[
+        "cap_and_util"]
+    )
+
+    new_argvalues = []
+    new_idlist = []
+    for i, argvalue_tuple in enumerate(argvalues):
+        args = dict(zip(argnames, argvalue_tuple))
+
+        try:
```

Comment:
This section shouldn't be needed as you have the required fields above on line 20